index.html

```diff
@@ -95,12 +95,12 @@ <h4>Wins</h4>
                     <div class="card_main_program">
                         <i class="fa-solid fa-user-plus"></i>
                         <h4>Deliver</h4>
-                        <p>With all the map of challenges, opportunities and process we activate execution!</p>
+                        <p>With the map of challenges, opportunities and process we activate execution!</p>
                     </div>
                 </div>
             </div>
             <button>Contact Us</button>
-            <p class="click_experience"><a href="">SEE THE WHOLE EXPERIENCE</a></p>
+            <p class="click_experience"><a href="">SEE THE EXPERIENCE</a></p>
```

Comment:
- [ ] kindly increase the size of the menu close icon and make it bolder so it will be more visible
![Screenshot (626)_LI](https://user-images.githubusercontent.com/65298048/203656610-140045ce-5a44-4666-bab5-3e79b5bd0f79.jpg)